frontend/public/components/events.jsx

```diff
@@ -196,24 +205,24 @@ export class EventsList extends React.Component {
         </div>,
       );
     } else {
-      selected.forEach((kind) => {
-        events.push(
-          <div key={kind}>
-            <span>
+      events.push(
```

Comment:
This shouldn't need to be an array anymore, right?

We should be able to go back to what was essentially the logic in 4.3, correct? We just need to update the filter method. I think this can be greatly simplified now.